release/nightly_tests/shuffle_data_loader/shuffle_data_loader_compute.yaml

```diff
@@ -0,0 +1,22 @@
+cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
+region: us-west-2
+
+max_workers: 0
+
+aws:
+    BlockDeviceMappings:
+        - DeviceName: /dev/sda1
+          Ebs:
+            VolumeSize: 500
+
+head_node_type:
+    name: head_node
+    instance_type: i3.4xlarge
+    resources: {"object_store_memory": 53687091200}
+
+worker_node_types:
+    - name: worker_node
+      instance_type: i3.4xlarge
+      min_workers: 0
```

Comment:
@clarkzinzow is this supposed to test a single node? I thought we should test multi node.

Reply:
I think that we should have a simple single-node test (smaller data scale e.g. 100M rows) and a large-scale multi-node test (4 nodes total, 4 trainers per node, 32 reducers total, 400M rows, 50 files).

Reply:
But if we had to choose, the large-scale multi-node test is more important.